project-koku/koku
# Use multi-arch container images for development

## Description

Instead of pulling different images from different registries and maintaining a separate Docker Compose file, use images that contain both architectures.

## Testing

1. Checkout Branch
2. `make docker-up-min`
3. Verify the `trino` and `hive-metastore` images match your system architecture

```
docker ps --format 'table {{ .ID }}\t{{ .Names }}\t{{ .Image }}\t{{ .State }}\t{{ .Label "architecture" }}\t{{ .Ports }}'
```
## Release Notes

```markdown
* Use multi-arch container images for local development #5045
```
